### ISSUE TITLE
Extension breaks RedirectorPage

### ISSUE THREAD

**Issue by AndrewHaine** [bigfork/silverstripe-simpleseo#2]
Line 35 in `SimpleSEOExtension.php` is breaking redirector pages, looks like this is a result of RedirectorPages not having a tab for `Root.Main.Metadata`.

Quick fix could be this inside of updateCMSFields as Redirector Pages have no metadata
```php
if( $this->owner->ClassName === RedirectorPage::class ) return;
```

**Comment by lozcalver** [comment]
I’d probably suggest making it more generic, like:

```php
public function updateCMSFields(FieldList $fields)
{
    $metadataHolder = $fields->fieldByName('Root.Main.Metadata');
    if (!$metadataHolder) {
        return;
    }

    // ... Requirements and other stuff

    $metadataFields = $metadataHolder->getChildren();
    // ... rest of method
}
```

That way if someone builds a custom page type that doesn’t have those fields for whatever reason it won’t break then

**Comment by AndrewHaine** [comment]
Fixed in #3
